# Created at 2022-08-29T21:32:48.497
org.testng.TestNGException: 
The factory method class task.FactoryClass.factoryMethod() threw an exception
	at org.testng.internal.FactoryMethod.invoke(FactoryMethod.java:223)
	at org.testng.internal.TestNGClassFinder.processFactory(TestNGClassFinder.java:174)
	at org.testng.internal.TestNGClassFinder.processMethod(TestNGClassFinder.java:138)
	at org.testng.internal.TestNGClassFinder.processClass(TestNGClassFinder.java:129)
	at org.testng.internal.TestNGClassFinder.<init>(TestNGClassFinder.java:67)
	at org.testng.TestRunner.initMethods(TestRunner.java:441)
	at org.testng.TestRunner.init(TestRunner.java:335)
	at org.testng.TestRunner.init(TestRunner.java:288)
	at org.testng.TestRunner.<init>(TestRunner.java:178)
	at org.testng.SuiteRunner$DefaultTestRunnerFactory.newTestRunner(SuiteRunner.java:639)
	at org.testng.SuiteRunner.init(SuiteRunner.java:225)
	at org.testng.SuiteRunner.<init>(SuiteRunner.java:115)
	at org.testng.TestNG.createSuiteRunner(TestNG.java:1349)
	at org.testng.TestNG.createSuiteRunners(TestNG.java:1325)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1167)
	at org.testng.TestNG.runSuites(TestNG.java:1099)
	at org.testng.TestNG.run(TestNG.java:1067)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.FactoryMethod.invoke(FactoryMethod.java:184)
	... 23 more
Caused by: java.lang.NullPointerException
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:149)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
	at java.base/java.io.FileReader.<init>(FileReader.java:60)
	at task.ContentReader.readAllLines(ContentReader.java:11)
	at task.FactoryClass.factoryMethod(FactoryClass.java:22)
	... 28 more


